ihatemoney/web.py

```diff
@@ -269,6 +269,11 @@ def home():
         session=session,
     )
 
+@main.route("/download-mobile-app", methods=["GET"])
+def downloadMobileApp():
```

Comment:
The convention we're using elsewhere on the project is to have `camel_case` for the method names. So this should be `download_mobile_app`. In general in python this is what's been used :-)